variables.tf

```diff
@@ -401,8 +401,14 @@ variable "s3_lifecycle_policy_transition_period" {
   default     = "30"
 }
 
-variable "apiary_extensions_version" {
-  description = "Version of the Apiary-Extensions module. Only for deployment with Ansible Playbook."
+variable "kafka_bootstrap_servers" {
```

Comment:
We are kind of abusing this to contain both the server info and also something which is more like `kafka_events_enabled`. On the other hand it means one less thing to configure. I can live with it, we just to need to be clear that the other kafka variables don't turn Kafka events on, this one does. So we need to document that here (which you've done) and also that it contains the list of bootstrap servers (which is missing currently). Or we need to introduce another variable to turn the Kafka events on. Thoughts from others?

Reply:
@massdosage - I agree in theory, but we are also using config vars like `SNS_ARN`, `RANGER_POLICY_MANAGER_URL`, etc, to do the same things for other metastore features.   So this is consistent with the current code.  I am ok with keeping it this way, and when we decide on a better (more explicit) way of turning on/off these features, do it consistently across the code.

Reply:
Well, we already have `enable_gluesync`, `enable_metadata_events` and others so I'd argue having an enabled flag for this is actually *more* consistent with what we already have in this file?

Reply:
main difference between gluesyn/sns and kafka is , we are not providing any service info, so just enable is sufficient for those cases.

for kafka and ranger we need to providing server info, if we add enable variable, we need 2 variables, which is redundant to accomplish the same.

Reply:
OK, so what is `enable_metadata_events` used for? From reading the description here I thought it was to enable the SNS listener?

Reply:
https://github.com/ExpediaGroup/apiary-data-lake/blob/master/k8s-readwrite.tf#L83
when we set enable_metadata_events, SNS_ARN(managed by terraform) environment variable is passed on to docker container

Reply:
OK, fine, let's go with this for now. At some point we might decide to refactor to have a different/more consistent way of enabling/disabling features etc.